parser: merge_stmt-v1

junghyun4425 opened the pull request:
수정된 사항
- yacc파일에서 리스트를 받을수 있게하기 위해 VM에 stmt_list를 추가
- yacc파일과 vm간 연동
- stmt만 구현되어, test코드가 에러발생하여 test코드 제거

namhyung commented:
어떤 오류가 발생하는지 정확히 보여줄 수 있나요? 제 생각에는 ParserContext에 vm이 없어서 nullptr 참조 때문에 죽는거 같은데 다른 문제가 더 있나요?

junghyun4425 committed:
parser: mergr_expr

junghyun4425 committed:
parser: Connected yacc file with vm

apus.y can make list of statement,
 and deliver StmtListPtr to vm object.

Now, only added statement part.

And must delete test/parset/testfiles,
 because of run_unit_test error.

namhyung commented:
ReturnStatement 생성자에서 Expression으로 garbage가 들어와서 포인터 참조 오류 발생.

junghyun4425 committed:
parser: data_decl connection
